Define headless CMS at start of article (#17455)
gatsbyjs/gatsby: docs/docs/headless-cms.md

---
title: What is a Headless CMS and how to source content from one
overview: true
---

> A *headless CMS* is content management software that enables Writers to produce and organize content, while providing Web Developers with structured data that can be displayed using a separate system on the front-end of a website or app.

Reviewer: ```suggestion
A *headless CMS* is content management software that enables writers to produce and organize content, while providing developers with structured data that can be displayed using a separate system on the front-end of a website or app.
```

I think this doesn't need to capitalized? And since the definition says it can separate the front end of a website _or_ app I think it could just say developers?

> A _headless CMS_ is content management software that enables Writers to produce and organize content, while providing Web Developers with structured data that can be displayed using a separate system on the front-end of a website or app.

Reviewer 2: ```suggestion
A _headless CMS_ is content management software that enables Writers to produce and organize content, while providing Web Developers with structured data that can be displayed using a separate system on the frontend of a website or app.
```
This is also a super nitpick, but updating to match the Style Guide / glossary: https://www.gatsbyjs.org/docs/glossary#frontend

> A traditional, monolithic CMS is responsible for both the backend management of content, and serving that content to end users. In contrast, a headless CMS is decoupled from frontend concerns, which frees Web Developers to build rich experiences for end users, using the best technology available.

Reviewer: ```suggestion
A traditional, monolithic CMS is responsible for both the backend management of content, and serving that content to end users. In contrast, a headless CMS is decoupled from frontend concerns, which frees developers to build rich experiences for end users, using the best technology available.
```

> Many content management systems (CMS) now support a “headless” mode, which is perfect for Gatsby sites.

Reviewer 2: ```suggestion
Many content management systems (CMS) now support a “headless” or “decoupled” mode, which is perfect for Gatsby sites.
```
